docker/build-tools/Dockerfile

```diff
@@ -43,28 +43,28 @@ ENV GOCOVMERGE_VERSION=b5bfa59ec0adc420475f97f89b58045c721d761c
 ENV GOGO_PROTOBUF_VERSION=v1.3.2
 ENV GOIMPORTS_VERSION=v0.1.0
 ENV BENCHSTAT_VERSION=9c9101da8316
-ENV GH_VERSION=2.0.0
+ENV GH_VERSION=2.5.2
 ENV GOLANG_PROTOBUF_VERSION=v1.27.1
 ENV GOLANG_GRPC_PROTOBUF_VERSION=v1.2.0
 # When updating the golangci version, you may want to update the common-files config/.golangci* files as well.
-ENV GOLANGCI_LINT_VERSION=v1.44.0
+ENV GOLANGCI_LINT_VERSION=v1.44.2
 ENV HADOLINT_VERSION=v2.8.0
 ENV HELM3_VERSION=v3.8.0
-ENV HUGO_VERSION=0.79.1
+ENV HUGO_VERSION=0.93.3
 ENV JB_VERSION=v0.3.1
 ENV JSONNET_VERSION=v0.15.0
 ENV JUNIT_MERGER_VERSION=adf1545b49509db1f83c49d1de90bbcb235642a8
-ENV K8S_CODE_GENERATOR_VERSION=1.23.0
+ENV K8S_CODE_GENERATOR_VERSION=1.23.4
 ENV K8S_TEST_INFRA_VERSION=2acdc6800510dd422bfd2a5d8c02aedc19d15f8d
 ENV KIND_VERSION=v0.12.0
-ENV KUBECTL_VERSION=1.23.1
+ENV KUBECTL_VERSION=1.23.4
 ENV PROTOC_GEN_GRPC_GATEWAY_VERSION=v1.8.6
 ENV PROTOC_GEN_SWAGGER_VERSION=v1.8.6
-ENV PROTOC_GEN_VALIDATE_VERSION=v0.6.1
-ENV PROTOC_VERSION=3.18.0
+ENV PROTOC_GEN_VALIDATE_VERSION=v0.6.7
+ENV PROTOC_VERSION=3.19.4
 ENV PROTOLOCK_VERSION=v0.14.0
 ENV PROTOTOOL_VERSION=v1.10.0
-ENV SHELLCHECK_VERSION=v0.7.1
+ENV SHELLCHECK_VERSION=v0.8.0
```

Comment:
This will probably break some of our scripts. I'll fix them before I merge in common-files.